main.py

```diff
@@ -19,7 +19,8 @@ def main():
             if choice == 'Q':
                 break
         except:
-            traceback.print_last()
+            print("\nYou have this book already.\n")
```

Comment:
This is not appropriate here. If your program experiences any error your user will see a 'You have this book already' message which is probably not what the problem is. Each function should handle errors as part of that function